### DISCUSSION TITLE
Mouse buttons for osu!taiko are configurable, allowing for 6 key gameplay

### DISCUSSION THREAD

**Discussion by Skidooskei** [ppy/osu#25721] [Ideas]
I believe in stable M1 and M2 are not changeable, therefore unless you had a third arm for the mouse, osu!taiko is limited to 4 keys. If these inputs are left configurable in lazer, players can change these to something like "sdf jkl" and use a playstyle like "dddkkk" (or something else with 6 keys), which is a new playstyle that would be much more powerful than any previous playstyle, and would essentially change the gamemode.

My suggestion is to remove M1 and M2 from the configuration settings to keep inputs consistent with stable. Thanks :)

(This is my first time posting something on GitHub so I hope I did it right!)

**Comment by peppy** [comment]
Lazer allows two bindings per drum key, as stable did (mouse + keyboard), but these are shared, so if you're holding one down and you press the next it will not register, as stable did.
